qemu/tests/valgrind_memalign.py

```diff
@@ -51,5 +51,6 @@ def valgrind_intall():
 
     error.context("Quit guest and check the process quit normally",
                   logging.info)
-    vm.destroy(gracefully=False)
+    vm.monitor.quit()
```

Comment:
Looks good, there's only one typo in the commit message ("makeing").

Reply:
Updating.